webapp/src/main/java/life/qbic/views/login/LoginHandler.java

```diff
@@ -37,25 +41,34 @@ public void handle(LoginLayout loginView) {
       registeredLoginView = loginView;
     }
     initFields();
-    resetMessages();
     addListener();
   }
+
   private void initFields() {
-    registeredLoginView.password.setHelperText("A password must be at least 8 characters");
-    registeredLoginView.password.setPattern(".{8,}");
-    registeredLoginView.password.setErrorMessage("Invalid Password");
-    registeredLoginView.email.setErrorMessage("Invalid Email");
+    resetErrorMessages();
+  }
+
+  private void resetErrorMessages() {
+    showDefaultError();
```

Comment:
Why do you show the error here to just hide it immediatly afterwards? 
```suggestion
```

Reply:
I originally had `setDefaultError()` and that did not change the visibility of the error message.
As I want to make sure that the state of the view is reset I not only hide the error but overwrite it with the default one.
The method `setError()` would not communicate if the error is shown afterwards.